components/common-go/tracing/tracing.go

```diff
@@ -100,7 +100,7 @@ func FromContext(ctx context.Context, name string) (opentracing.Span, context.Co
 
 // ApplyOWI sets the owner, workspace and instance tags on a span
 func ApplyOWI(span opentracing.Span, owi logrus.Fields) {
-	for _, k := range []string{log.OwnerField, log.WorkspaceField, log.InstanceField} {
+	for _, k := range []string{log.OwnerField, log.WorkspaceField, log.InstanceField, log.ProjectField, log.TeamField} {
```

Comment:
Not necessarily in-scope for this PR, but: We started writing up some naming conventions for tracing, incl. [for OWI](https://www.notion.so/gitpod/Naming-conventions-af9084c4743349089cf2450a3c860733#5c8ee0a476904a9aa2abc637340aa747). (just extended it for projectId/teamId :innocent: )